manifests/claudie/ansibler.yaml

```diff
@@ -3,16 +3,19 @@ kind: Deployment
 metadata:
   name: ansibler
   labels:
-    app: ansibler
```

Comment:
Please double-check that we can safely remove the old labels without anything breaking